tests/pom.xml

```diff
@@ -341,6 +339,7 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
       <properties>
         <hono.kafka.disabled>false</hono.kafka.disabled>
         <hono.kafka.bootstrap.servers>kafka.hono:9092</hono.kafka.bootstrap.servers>
+        <request.timeout>4000</request.timeout>
```

Comment:
`request.timeout` is also getting set with the `ci_environment` (value `3000`), but this `kafka` profile being defined after it in the pom seems to mean it has precedence. So that should work. The question is if 4000 is even enough for CI environments, but we can test this when enabling the Kafka test run.